decompiler/structures/pseudo/expressions.py

```diff
@@ -250,6 +250,20 @@ def copy(self) -> ExternFunctionPointer:
         return ExternFunctionPointer(self.value, self._type.copy(), self.tags)
 
 
+class NotUseableConstant(Constant):
```

Comment:
Please add tests for this new class (see test for constant).

- Since this was also a problem with the z3-converter, a test here would also be beneficial.

Reply:
Done, two catches:
- OverflowError (not covered by Deadpath, yields this issue)
- ValueError (covered by Deadpath, value will be skipped for z3)